openapi-core/src/main/java/io/ballerina/openapi/core/GeneratorUtils.java

```diff
@@ -750,7 +748,7 @@ public static void validateRequestBody(Set<Map.Entry<String, PathItem>> paths) t
         }
 
         if (!errorList.isEmpty()) {
-            StringBuilder errorMessage = new StringBuilder("OpenAPI definition has errors: " + DOUBLE_LINE_SEPARATOR);
+            StringBuilder errorMessage = new StringBuilder("OpenAPI definition has errors: " + System.lineSeparator());
```

Comment:
To maintain the consistency. 

```suggestion
            StringBuilder errorMessage = new StringBuilder("OpenAPI definition has errors: " + LINE_SEPARATOR);
```